sdk/python/feast/infra/offline_stores/snowflake.py

```diff
@@ -453,6 +463,41 @@ def persist(self, storage: SavedDatasetStorage):
     def metadata(self) -> Optional[RetrievalMetadata]:
         return self._metadata
 
+    def supports_remote_storage_export(self) -> bool:
```

Comment:
this is inconsistent with BQ - does this method represent "whether this offline store supports remote storage export" (in which case it should just be `True`) or "whether remote storage export will work for this instance" (in which case it should be this kind of check)

Reply:
I think this should be the latter. Updated.